test/spec/modules/mediasquareBidAdapter_spec.js

```diff
@@ -24,6 +24,43 @@ describe('MediaSquare bid adapter tests', function () {
       code: 'publishername_atf_desktop_rg_pave'
     },
   }];
+  var VIDEO_PARAMS = [{
+    adUnitCode: 'banner-div',
+    bidId: 'aaaa1234',
+    auctionId: 'bbbb1234',
+    transactionId: 'cccc1234',
+    mediaTypes: {
+      video: {
+        context: 'instream',
+        playerSize: [640, 480],
+        mimes: ['video/mp4'],
+      }
+    },
+    bidder: 'mediasquare',
```

Comment:
The bidder data need to be in a bids array. Please validate your params and setting. I'm not seeing any responses using this test data.

Reply:
Sorry but I don't get what you mean here. What data are you refering has need to be in a bids array ? When you say you are not seing any responses using this data, you mean using a real call ?

Reply:
@GLStephen I don't know if you've seen my previous comment. Could you please be more specifc because I really don't get what you meant.

Reply:
The test params, as far as I can tell are in an invalid format.

```
[{
    adUnitCode: 'banner-div',
    bidId: 'aaaa1234',
    auctionId: 'bbbb1234',
    transactionId: 'cccc1234',
    mediaTypes: {
      native: {
        title: {
          required: true,
          len: 80
        },
      }
    },
    bidder: 'mediasquare',
    params: {
      owner: 'test',
      code: 'publishername_atf_desktop_rg_pave'
    },
  }]
```
the bidder should be in a bidders array, I was just asking you to double check their format
```
[{
    adUnitCode: 'banner-div',
    bidId: 'aaaa1234',
    auctionId: 'bbbb1234',
    transactionId: 'cccc1234',
    mediaTypes: {
      native: {
        title: {
          required: true,
          len: 80
        },
      }
    },
    bidders: [{
    bidder: 'mediasquare',
    params: {
      owner: 'test',
      code: 'publishername_atf_desktop_rg_pave'
    }
    }]
  }]
```

Reply:
@GLStephen I just checked how the object is build while calling the buildRequests() methods and it confirms that my bidRequest objects is a valid one. It also matches the builder adapter documentation here : https://docs.prebid.org/dev-docs/bidder-adaptor.html#building-the-request which give this example for a validBidRequests[] 
```
[{
  adUnitCode: "test-div"
  auctionId: "b06c5141-fe8f-4cdf-9d7d-54415490a917"
  bidId: "22c4871113f461"
  bidder: "rubicon"
  bidderRequestId: "15246a574e859f"
  bidRequestsCount: 1
  bidderRequestsCount: 1
  bidderWinsCount: 0
  userId: {...}
  schain: {...}
  mediaTypes: {banner: {...}}
  params: {...}
  src: "client"
  transactionId: "54a58774-7a41-494e-9aaf-fa7b79164f0c"
}]
```
I think you might have mixed up with the bidderRequest object.